pennylane/templates/embeddings/displacement.py

```diff
@@ -100,6 +100,12 @@ def circuit(feature_vector):
     num_wires = AnyWires
     grad_method = None
 
+    @classmethod
+    def _unflatten(cls, data, metadata):
+        new_op = cls.__new__(cls)
```

Comment:
why was this necessary?

Reply:
Easiest way to bypass the preprocessing done in `__init__`.  We already have the data, so we don't need to cast and stack it. `op.data` is not a valid `features`, so we have to do something else instead.